### PR TITLE
Expose CloudFormation capabilities 

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Expose capabilities as independent checkboxes when deploying, and save them to deployment settings
- Remove very weird spacers from the deployment window that made the template parameters box look compressed

## Related Issue(s)
#1394 

## Screenshots (if appropriate)
Screen Shot 2020-04-08 at 4 42 55 PM

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/CreateCapabilities.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        true
+    ),
+    NAMED_IAM(
+        "CAPABILITY_NAMED_IAM",
+        message("cloudformation.capabilities.named_iam"),
+        message("cloudformation.capabilities.named_iam.toolTipText"),
+        true
+    ),
+    AUTO_EXPAND(
+        "CAPABILITY_AUTO_EXPAND",
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/CreateCapabilities.kt:23]: Can we use the actual enum here? (Capability.CAPABILITY_AUTO_EXPAND.value)

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/CreateCapabilities.kt:23]: Or better yet, the enum itself and convert to the value string when we need it?

Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/deploy/CreateCapabilities.kt:23]: `value` is actually private, but the toString works, changed. This is really the point where we need it so I don't think we need to store anything other than the string.

**Comment by kiiadi** [comment]
@bryceitoc9 @justinmk3 @awschristou  how are we handling this in vscode?

**Pull request merged**